ui/ozone/platform/starboard/BUILD.gn

```diff
@@ -33,6 +35,7 @@ source_set("starboard") {
   ]
 
   deps = [
+    "//starboard($starboard_toolchain)",
```

Comment:
@niranjanyardi you'll probably want to start thinking about if we want people to depend on //starboard or //starboard:starboard_group after your work on the hermetic toolchain is complete—I'll assume we'll need to do the latter

Reply:
latter +1

Reply:
Done.

Reply:
Oh actually it looks like this is the cause of the new build errors:

```
ERROR at //ui/ozone/platform/starboard/gl_ozone_egl_starboard.cc:19:11: Can't include this header from here.
#include "starboard/egl.h"
          ^--------------
The target:
  //ui/ozone/platform/starboard:starboard
is including a file from the target:
  //starboard:starboard_headers_only

It's usually best to depend directly on the destination target.
In some cases, the destination target is considered a subcomponent
of an intermediate target. In this case, the intermediate target
should depend publicly on the destination to forward the ability
to include headers.

Dependency chain (there may also be others):
  //ui/ozone/platform/starboard:starboard -->
  //starboard:starboard_group --[private]-->
  //starboard:starboard -->
  //starboard/common:common -->
  //starboard:starboard_headers_only
```

Is there something else I need to do beside swap that to starboard_group?